pageStart.js

```diff
@@ -0,0 +1,26 @@
+import { Selector, t } from 'testcafe';
+
+export default class StartPage {
+constructor(){
+this.themeTitle = /anime/g;
+this.storeTitle = /communityrecommendations/g;
+this.yourStore = Selector('#foryou_tab .pulldown_desktop');
+this.community = Selector('#foryou_flyout a');
+this.Categories = Selector('#genre_tab .pulldown_desktop');
+this.theme = Selector('[data-genre-group="themes"] .popup_menu_item');
+
+    }
+async themesClicker(link){
```

Comment:
it is a common practice to used verbs in the names of function to describe what they do. e.g. clickTheme, selectTheme, etc.